tests/data_files/cert_example_wildcard.crt

```diff
@@ -1,77 +1,20 @@
-Certificate:
```

Comment:
Don't we want to have such information about the new certificate here?

Reply:
We can, but what's the added value?
Frankly I don't think this should block the PR.